examples/web-wagmi/src/publications/components/Medias.tsx

```diff
@@ -0,0 +1,36 @@
+import { Post, Comment, Media } from '@lens-protocol/react-web';
+
+import { isSupportedImageMedia } from '../../utils/media';
+import { resolveFullResourceUrl } from '../../utils/resolveFullResourceUrl';
+
+type MediaItemProps = {
+  media: Media;
+};
+
+function MediaItem({ media }: MediaItemProps) {
+  if (isSupportedImageMedia(media)) {
+    const uri = resolveFullResourceUrl(media.url);
+
+    return <img src={uri} alt={media.altTag || ''} width="45%" />;
+  }
+
+  return null;
+}
+
+type MediaProps = {
+  publication: Post | Comment;
+};
+
+export function Medias({ publication }: MediaProps) {
```

Comment:
nit: "media" is such an annoying word. I think it's already plural but in common english it has been used as singular in it's plural form. 🤷

Reply:
right, thanks!